pages/donnees_agriculture-alimentation.html

```diff
@@ -294,16 +294,9 @@ <h3 id="foret">Forêt</h3>
                                     <li><a
                                             href="https://www.data.gouv.fr/fr/datasets/donnees-brutes-de-l-inventaire-forestier/">Données
                                             brutes de l’inventaire forestier</a></li>
-                                    <li><a href="https://www.data.gouv.fr/fr/datasets/bd-foret-r/">BD Forêt</a></li>
-                                    <li><a
-                                            href="https://www.data.gouv.fr/fr/datasets/forets-publiques-diffusion-publique-1/">Forêts
-                                            publiques</a></li>
-                                    <li><a
-                                            href="https://www.data.gouv.fr/fr/datasets/parcelles-forestieres-publiques-diffusion-publique/">Parcelles
-                                            forestières publiques</a></li>
-                                    <li><a
-                                            href="https://www.data.gouv.fr/fr/datasets/reserves-biologiques-diffusion-publique/">Réserves
-                                            biologiques</a></li>
+                                    <li><a 
```

Comment:
WHat means "BD"? Sorry for newbie question... Might be good to be more explicit though for newbies!

Reply:
It stands for "Base de Données" its a trademark of the IGN "BD Forêt®" https://www.data.gouv.fr/fr/datasets/bd-foret-r/